### DISCUSSION TITLE
Fabric --setup

### DISCUSSION THREAD

**Discussion by dsaldivar** [danielmiessler/Fabric#112] [Q&A]
Hello, Thanks in advance for your assistance.  First let me say I'm running on a Windows 10 laptop and using Powershell through VS to do all the installation setups of fabric.  I apologize but I'm a novice when it comes to understanding unix commands and its environment. I have been able to limp along up to the point of issuing the command fabric --setup.  In Powershell it returns an error ObjectNotFound.  I'm not sure if it matters but when I ran the setup.sh script file , all of the dependencies installed fine, however, at the end when it went to install the project fabric, it indicated that all of the alias' didn't exist.  In my inquire to ChatGpt about this problem it responded that since I was running on Windows those alias' MAY not be necessary. So I'm just checking to see if not having these alias' would cause the problem.  Anyway, any help would be appreciated. Thank you.

**Comment by Muuver** [comment]
Hello, I'm having the same problem.
I'm new to terminal commands as well, but I'm on a mac.
I followed the install instructions up to step 6, but while running step 5 I got this returned:

```
Installing python dependencies
./setup.sh: line 5: poetry: command not found
/Users/myname/.bashrc does not exist.
/Users/myname/.zshrc does not exist.
/Users/myname/.bash_profile does not exist.
Please close this terminal window to have new aliases work.
```

So I figured from the last line in that error that restarting terminal would fix things up (as per step 6) so I restarted terminal, went back to the /fabric directory, then I entered the new command `fabric --setup` into terminal and got this error:
`-bash: fabric: command not found`
Then I tried just '--setup' and got a similar error:
`-bash: --setup: command not found`

Perhaps there's an error on line 5 of the setup file?

**Reply by dsaldivar** [reply]
Hi Muuver,
I'm not sure but the setup.sh script file contains commands that are easily executed if you are operating in a Unix environment.  We are both using terminals in different OSs. I'm not familiar enough Unix to decipher how to do in my OSs terminal (Powershell) however I tried to run the script file using Git Bash and the same message returns of alias' does not exist. I've been using Chatgpt to help me troubleshoot this and get me past this point but still no success.  Let me know if you have found out anything. Thanks

**Comment by danielmiessler** [comment]
Are you still having this issue?

I'd try deleting the repo and starting over. The installer is far cleaner now.

**Reply by dsaldivar** [reply]
No. Thanks Daniel.  I haven't tried reinstalling using VSC with Powershell terminal. I was able to get it to get through the entire quickstart setup after installing the extension Windows Subsystem for Linux. This provided me with an Ubunta shell terminal in VSC for unix compatibility.
